src/picongpu/submit/titan-ornl/pythonOnRhea.profile.example

```diff
@@ -36,17 +36,21 @@ source $PROJWORK/$proj/python-venv/rhea/bin/activate
 #
 # now run for setting up the python virtual environment
 #   mkdir -p $PROJWORK/$proj/python-venv
-#   cd $PROJWORK/$proj
+#   cd $PROJWORK/$proj/python-venv
```

Comment:
I corrected it, unfortunately the old line was wrong: the command below `virtualenv rhea` creates a new directory `rhea` in the *current* directory which is in all other places assumed to be `$PROJWORK/$proj/python-venv/rhea`.